Application/ControlMsrE2/ControlMsrE2.c

```diff
@@ -0,0 +1,248 @@
+/** @file
+  Control CFGLock BIOS Option
+
+Copyright (c) 2020, Brumbaer. All rights reserved.<BR>
+This program and the accompanying materials
+are licensed and made available under the terms and conditions of the BSD License
+which accompanies this distribution.  The full text of the license may be found at
+http://opensource.org/licenses/bsd-license.php
+
+THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+
+**/
+
+#include "ControlMsrE2.h"
+
+#define CONTEXTS_MAX 8
+
+VOID
+WalkListHeaders (
```

Comment:
How about IterateListHeaders? Or even more meaningful names would be better (cannot say what to call exactly as I have not completely read the code yet)